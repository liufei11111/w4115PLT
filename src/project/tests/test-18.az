{* option declaration *}

Option o2 = { a="" };
Option o3 = { a="a" };
String str;
str = "str";
Option o4 = { a=str };
Option o5 = { a=str, b = "b"};