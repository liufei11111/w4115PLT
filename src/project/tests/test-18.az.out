stmts:

Option o2 = {a = ""};
Option o3 = {a = "a"};
string str;
str = "str";
Option o4 = {a = str};
Option o5 = {a = str, b = "b"};

fdecls:



